packages/desktop-client/src/components/mobile/transactions/TransactionListWithBalances.jsx

```diff
@@ -67,7 +67,7 @@ export function TransactionListWithBalances({
   onSearch,
   onLoadMore,
   onOpenTransaction,
-  onRefresh,
+  onRefresh = () => {},
```

Comment:
This should be undefined as default because there's logic below which checks for this prop. Setting an empty function will introduce a regression.